server/meca/file-generators/article.test.js

```diff
@@ -27,6 +28,7 @@ describe('Article XML generator', () => {
   it('generates expected XML', async () => {
     let xml = await generateXml(sampleManuscript)
     xml = obfuscateEmail(xml)
-    expect(xml).toMatchSnapshot()
+    if (config.get('server.api.secret')) expect(xml).toMatchSnapshot()
```

Comment:
I'd suggest putting this into a separate test so it's not hidden behind an `if` statement.

We can mock out `config` so that there is a secret set for the test.

Reply:
This should also mean there's no need for obsolete snapshots.